admin-part/src/main/kotlin/Plugin.kt

```diff
@@ -144,6 +146,39 @@ class Plugin(
                 ActionResult(StatusCodes.OK, "")
             } ?: ActionResult(StatusCodes.NOT_FOUND, "Active session '$sessionId' not found.")
         }
+        is ExportCoverage -> runCatching {
+            val probesByteArray = ByteArrayOutputStream().use { outputStream ->
+                val executionDataWriter = ExecutionDataWriter(outputStream)
+                val classBytes = state.coverContext().classBytes
+                val allFinishedScopes = state.scopeManager.byVersion(buildVersion, true)
+                allFinishedScopes.flatMap { finishedScope ->
+                    finishedScope.data.sessions.flatMap { it.probes }
+                }.forEach { execClassData ->
+                    executionDataWriter.visitClassExecution(
+                        ExecutionData(
+                            classBytes[execClassData.className]?.crc64() ?: execClassData.id(),
+                            execClassData.className,
+                            execClassData.probes.toBooleanArray()
+                        )
+                    )
+                }
```

Comment:
```suggestion
                }.forEach {
                with(it){
                    executionDataWriter.visitClassExecution(
                        ExecutionData(
                            classBytes[className]?.crc64() ?: id(),
                            className,
                            probes.toBooleanArray()
                        )
                    )
                }
             }
```
what do u think about it , does it look better ?

Reply:
No! looks so weird. 
Kotlin already has 
/**
 * Performs the given [action] on each element and returns the collection itself afterwards.
 */
@SinceKotlin("1.1")
public inline fun <T, C : Iterable<T>> C.onEach(action: (T) -> Unit): C {
    return apply { for (element in this) action(element) }
}

construction.